lib/config/config-initializer.js

```diff
@@ -573,7 +553,16 @@ function promptUser() {
                 earlyAnswers.styleguide = "airbnb-base";
             }
 
-            const config = ConfigOps.merge(processAnswers(earlyAnswers), getConfigForStyleGuide(earlyAnswers.styleguide));
+            const config = processAnswers(earlyAnswers);
+
+            if (Array.isArray(config.extends)) {
+                config.extends.push(earlyAnswers.styleguide);
+            } else if (config.extends) {
+                config.extends = [config.extends, earlyAnswers.styleguide];
+            } else {
+                config.extends = earlyAnswers.styleguide;
```

Comment:
Maybe it would be better to use an array here so that it's more clear how the user can extend additional configs if needed.

```suggestion
                config.extends = [earlyAnswers.styleguide];
```